src/controller/ai_controller.c

```diff
@@ -1339,29 +1816,28 @@ void ai_controller_create(controller *ctrl, int difficulty, sd_pilot *pilot, int
     }
     a->blocked = 0;
     a->thrown = 0;
-    a->queued_tactic = 0;
+    
     vector_create(&a->active_projectiles, sizeof(object*));
     a->pilot = pilot;
 
-    // pilot prefs manually until we start reading them from binary
+    // set pilot prefs manually until we start reading them from binary
     populate_pilot_prefs(pilot, pilot_id);
-    DEBUG("pilot %d", pilot_id);
-    DEBUG("att_normal %d", pilot->att_normal);
-    DEBUG("att_def %d", pilot->att_def);
-    DEBUG("att_hyper %d", pilot->att_hyper);
-    DEBUG("att_jump %d", pilot->att_jump);
-    DEBUG("att_sniper %d", pilot->att_sniper);
-    DEBUG("ap_special %d", pilot->ap_low);
-    DEBUG("ap_middle %d", pilot->ap_middle);
-    DEBUG("ap_high %d", pilot->ap_high);
-    DEBUG("ap_jump %d", pilot->ap_jump);
-    DEBUG("ap_throw %d", pilot->ap_throw);
-    DEBUG("ap_special %d", pilot->ap_special);
+
+    // set initial tactical state
+    tactic_state *tactic = malloc(sizeof(tactic_state));
```

Comment:
Would be safer to use calloc or malloc+memset here in case the struct gains more stuff